packages/synapse-react-client/src/components/AccountLevelBadge/AccountLevelBadge.tsx

```diff
@@ -1,67 +1,66 @@
+import { Link, Tooltip, Typography } from '@mui/material'
 import React from 'react'
 import { ReactComponent as Certified } from '../../assets/icons/account-certified.svg'
-import { ReactComponent as Registered } from '../../assets/icons/account-registered.svg'
 import { ReactComponent as Validated } from '../../assets/icons/account-validated.svg'
-import { SynapseConstants } from '../../utils'
-import { useGetUserBundle } from '../../synapse-queries/user/useUserBundle'
-import { ErrorBanner } from '../error/ErrorBanner'
 
-export type AccountLevelBadgeProps = {
-  userId: string
+export type AccountLevelBadgeType = 'certified' | 'validated'
+type AccountLevelBadgeConfig = {
+  label: string
+  description: string
+  tooltipText: string
+  icon: React.ReactElement
 }
 
-export const accountLevelRegisteredLabel: string = 'Registered'
-export const accountLevelCertifiedLabel: string = 'Certified'
-export const accountLevelVerifiedLabel: string = 'Validated'
+export const accountLevelBadgeConfig: Record<
+  AccountLevelBadgeType,
+  AccountLevelBadgeConfig
+> = {
+  certified: {
+    label: 'Certified',
+    description: 'Has access to normal functionality.',
+    tooltipText:
+      'This user has passed Sage Certification, which means they can upload data and create new projects and tables.',
+    icon: <Certified />,
+  },
+  validated: {
+    label: 'Validated',
+    description: 'Identity has been manually verified.',
+    tooltipText:
+      'This user is Validated. Their identity and credentials have been manually verified by Sage Bionetworks.',
+    icon: <Validated />,
+  },
+}
 
-const CERTIFICATION_AND_VERIFICATION_BUNDLE_MASK =
-  SynapseConstants.USER_BUNDLE_MASK_IS_CERTIFIED |
-  SynapseConstants.USER_BUNDLE_MASK_IS_VERIFIED
+export type AccountLevelBadgeProps = {
+  badgeType: AccountLevelBadgeType
+}
 
 export const AccountLevelBadge: React.FunctionComponent<
   AccountLevelBadgeProps
-> = ({ userId }: AccountLevelBadgeProps) => {
-  const {
-    data: userBundle,
-    isLoading,
-    error,
-  } = useGetUserBundle(userId, CERTIFICATION_AND_VERIFICATION_BUNDLE_MASK)
-
-  if (isLoading) {
-    return <></>
-  }
-  if (error) {
-    return <ErrorBanner error={error} />
-  }
-
-  let accountLevelString: string = accountLevelRegisteredLabel
-  let icon = <Registered />
-  if (userBundle?.isCertified) {
-    accountLevelString = accountLevelCertifiedLabel
-    icon = <Certified />
-  }
-  if (userBundle?.isVerified) {
-    accountLevelString = accountLevelVerifiedLabel
-    icon = <Validated />
-  }
+> = ({ badgeType }: AccountLevelBadgeProps) => {
+  const badgeConfig = accountLevelBadgeConfig[badgeType]
   return (
     <div className={'AccountLevelBadge cardContainer'}>
```

Comment:
Kind of picky and maybe out-of-scope for this ticket, so feel free to ignore--consider checking if this looks closer to the design if we drop the `.cardContainer` styling and use a MUI `Card` component.

I think that the `.cardContainer` class styles and our design system have drifted, but I'd be worried to change `.cardContainer` in this PR since it is used in many places.

If that's an easy change, maybe we follow up with a ticket to remove the style and replace it with the component.

Reply:
Good call! I updated this component to use the MUI `Card` and opened [this ticket](https://sagebionetworks.jira.com/browse/SWC-6652) to track the work elsewhere.